linux/step01_ubuntu2204_ice_deps.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+ICEVER=${ICEVER:-ice36}
+
+echo $ICEVER
+# Ice installation
+if [ "$ICEVER" = "ice36" ]; then
+    #start-recommended
+    apt-get update && \
+    apt-get install -y -q \
+        build-essential \
+        db5.3-util \
+        libbz2-dev \
+        libdb++-dev \
+        libdb-dev \
+        libexpat-dev \
+        libmcpp-dev \
+        libssl-dev \
+        mcpp \
+        zlib1g-dev
```

Comment:
Unrelated to this PR but I am unsure all of these packages are required at runtime.

Reply:
True, installing dev packages is strange. I'll remove them and see what happens.

Reply:
For some dependencies it must be legacy when we were building from source in omero-install